javalin/src/main/java/io/javalin/websocket/WsCloseStatus.kt

```diff
@@ -0,0 +1,121 @@
+package io.javalin.websocket
+
+/**
+ * WebSocket connection close codes.
+ *
+ * @see <a href="https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent/code">MDN Web Docs</a> for a brief description of each code.
+ * @see <a href="https://tools.ietf.org/html/rfc6455#section-7.4.1">RFC 6455, Section 7.4.1</a> for more information.
+ */
+enum class WsCloseStatus(val code: Int) {
+
+    /**
+     * The connection successfully completed the purpose for which it was created.
+     */
```

Comment:
Perhaps a weird thing to complain about, but having these elaborate descriptions of WebSocket close status codes doesn't really feel appropriate. In `HttpStatus`, which is probably used way more, we have no such descriptions.

Reply:
I think it kinda makes sense, because these codes are less known, so we can inform people about their purpose. Even in our tests, these codes are pretty much just random numbers without a meaning.

Also:
* https://github.com/javalin/javalin/issues/2016

Reply:
Fair enough, it's better to add more than to remove 😄